flow.md

```diff
@@ -0,0 +1,84 @@
+# Work-flow
+
+## Repositories
+| Type | Description |
+| ---| ---| ---| 
+| NETN FOM  | Main Repository for the entire FOM including all related FOM Modules.|
```

Comment:
The name of the repo is actually NETN-FOM. Also, looking at this file from regular GitHub UI, there seems to be some markdown problems in this table.

Reply:
Fixed.